lib/diagnostic.js

```diff
@@ -19,15 +19,18 @@
 //
 //     `true` or `false`
 //
-let access = null; // replace null with your response
+let access = true; // replace null with your response
 //
 
 // Which scope(s) may access the variable `laserQuantity`? Choose one or many
 // of the following:
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-let scopes = [/* your response here */];
+let scopes = ['alderaan', 'deathStar', 'fire', 'global'];
```

Comment:
just `deathstar` and `fire` because `laserQuantity` is defined within the scope of deathstar and fire is also so it has access to deathstar's scope